sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -47,11 +47,12 @@ export function buildQueueOptions(queue: CreateQueueOptions): InternalQueueOptio
     EnablePartitioning: getStringOrUndefined(queue.enablePartitioning),
     ForwardDeadLetteredMessagesTo: getStringOrUndefined(queue.forwardDeadLetteredMessagesTo),
     ForwardTo: getStringOrUndefined(queue.forwardTo),
-    UserMetadata: getStringOrUndefined(queue.userMetadata)
+    UserMetadata: getStringOrUndefined(queue.userMetadata),
+    EnableExpress: getStringOrUndefined(queue.enableExpress)
   };
 }
 
-/**
+/**ß
```

Comment:
Not sure how this got added...

```suggestion
/**
```